apps/web/composables/useHomepageData/types.ts

```diff
@@ -22,7 +22,9 @@ export type HeroItem = {
   actualBackgroundSize: SizeKey;
 };
 
-export type HeroCarouselProps = {
-  background: { image: string; sizes: Sizes };
-  hero: HeroItem[];
+export type MediaItem = {
```

Comment:
```suggestion
export type MediaCardProps = {
```
All props types should have a `Props` suffix. Applies to the `HeroItem`(which should probably be `HeroContentProps` now) as well.